charts/kubeclarity/values.yaml

```diff
@@ -316,8 +316,8 @@ kubeclarity-sbom-db:
       memory: "20Mi"
       cpu: "10m"
     limits:
-      memory: "100Mi"
-      cpu: "100m"
+      memory: "1Gi"
```

Comment:
Were the memory spikes disappeared after gzip? If so, why do we need this?

Reply:
To be on the safe side, low memory limits could result with a kernel oom killer for nothing in case of spikes of memory that we just had. It doesn't required more from the nodes, it just to prevent memory leakage over time.